libs/sdk-backend-tiger/src/backend/tigerSpecificFunctions.ts

```diff
@@ -349,6 +352,17 @@ export type TigerSpecificFunctions = {
     createCSPDirective?: (requestData: ICSPDirective) => Promise<ICSPDirective>;
     updateCSPDirective?: (directiveId: string, requestData: ICSPDirective) => Promise<ICSPDirective>;
     deleteCSPDirective?: (directiveId: string) => Promise<void>;
+    getWorkspaceCustomAppSettings?: (
```

Comment:
Plase add some interface docs for all the methods - what is the pupose / for what is it used for (eg concrete use case in our applications), ideally with some examples (eg. how values could look like).

Reply:
I added some docs. Please take a look. thank you.